search/controllers/api/__init__.py

```diff
@@ -77,6 +77,40 @@ def search(params: MultiDict) -> Tuple[Dict[str, Any], int, Dict[str, Any]]:
     logger.debug('Got document set with %i results', len(document_set.results))
     return {'results': document_set, 'query': q}, status.HTTP_200_OK, {}
 
+def classic_query(params: MultiDict) -> Tuple[Dict[str, Any], int, Dict[str, Any]]:
+    """
+    Handle a search request from the Clasic API. Maps old rquest 
```

Comment:
Nit: summary should fit on one line. Next block can be multiline. Not a blocker.